### PR TITLE
chore(deps): setup dependabot

### PR DESCRIPTION
**Pull request by mmorel-35**
Provides dependabot configuration to keep npm and github-actions up-to-date.
This is based on [kuma configuration](https://github.com/kumahq/kuma/blob/master/.github/dependabot.yml) 

Signed-off-by: Matthieu MOREL <matthieu.morel35@gmail.com>

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
```
Comment (kleinfreund) [inline .github/dependabot.yml:3]: **Issue**: This project uses yarn instead of npm.

```suggestion
  - package-ecosystem: "yarn"
```

**Review by kleinfreund — CHANGES_REQUESTED** [review]
Hey, thanks, you’re getting ahead of us with this one. I had intentions to set-up dependabot as well. One thing that I want to do before this goes in is pinning the dependencies by hand. I know dependabot would do this on its first PR, but I want to ensure this works by hand.

File: .github/dependabot.yml (hunk 1 above)
Comment (mmorel-35) [reply .github/dependabot.yml:3]: Take a look over here : https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem 

package-ecosystem for yarn is also npm

Comment (kleinfreund) [reply .github/dependabot.yml:3]: Ah, yeah I remember I ran into this before. Certainly very confusing.

**Review by kleinfreund — APPROVED** [review]

**Pull request merged**